examples/src/utils/fixtures.js

```diff
@@ -3,7 +3,7 @@
 const demoProject = {
     "https://secure.gooddata.com": "k26dtejorcqlqf11crn6imbeevp2q4kg",
     "https://staging3.intgdc.com": "kytra720hke4d84e8ozohoz7uycn53mi",
-    "https://staging2.intgdc.com": "ws7pxsamkx8o0t1s7kfvkj5o41uwcmqg",
+    "https://staging2.intgdc.com": "exveshu0intwbr4nk6ao8ft6vesjiqus",
```

Comment:
why do we change project id on staging2?

Reply:
fixed